code/game/machinery/syndicatebomb.dm

```diff
@@ -98,7 +98,7 @@
 
 /obj/machinery/syndicatebomb/examine(mob/user)
 	. = ..()
-	. += {"A digital display on it reads "[seconds_remaining()]"."}
+	// . += {"A digital display on it reads "[seconds_remaining()]"."} Skyrat change - commented out to make people fear it more.
```

Comment:
should be SKYRAT EDIT: - Commented out for fear factor

Reply:
or along those lines; it just needs to include 'SKYRAT EDIT' so we can ctrl F it

Reply:
Done